protos/sources/vega/commands/v1/transaction.proto

```diff
@@ -84,7 +96,7 @@ message Transaction {
 
   // A version of the transaction, to be used in the future in case changes are implemented
   // to the Transaction format.
-  uint32 version = 2000;
+  TxVersion version = 2000;
```

Comment:
I'm wondering how backward compatible is that? Is a transaction create with an old wallet still able to be decoded with this version of the core?

Reply:
It seems to work